ui/raidboss/data/06-ew/raid/p3n.txt

```diff
@@ -0,0 +1,110 @@
+## Pandaemonium: Asphedolos, the Third Circle
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 100000 jump 0
+
+# -ii 669F 66A7 66A8 66A0 66A1 66AF 66B0 66B3 6D0C 6DDC 6EDD
+# -ic "Liturgic Bell"
+# -it Phoinix
+# -p 66AD:118 66A3: 200
+
+0.0 "--sync--" sync /Engage!/ window 0,1
+9.1 "--sync--" sync / 14:[^:]*:Phoinix:(6696|6698):/ window 9.1,5
+13.8 "Experimental Fireplume" sync / 1[56]:[^:]*:Phoinix:(6696|6698):/ window 13.8,5
+19.0 "Fireplume (In/Out)" #sync / 1[56]:[^:]*:Phoinix:(6697|6699):/
+31.0 "Experimental Fireplume" sync / 1[56]:[^:]*:Phoinix:(6696|6698):/
+37.1 "Fireplume (Out/In)" #sync / 1[56]:[^:]*:Phoinix:(6697|6699):/
+49.2 "Scorched Exaltation" sync / 1[56]:[^:]*:Phoinix:66B8:/
+61.4 "Heat of Condemnation" sync / 1[56]:[^:]*:Phoinix:66B2:/ window 61.4,10
+75.6 "Darkened Fire" sync / 1[56]:[^:]*:Phoinix:6EDA:/
+85.7 "--sync--" sync / 1[56]:[^:]*:Phoinix:6EDC:/
+86.1 "Brightened Fire #1" #sync / 1[56]:[^:]*:Phoinix:6EDD:/
+87.2 "Brightened Fire #2" #sync / 1[56]:[^:]*:Phoinix:6EDD:/
+88.4 "Brightened Fire #3" #sync / 1[56]:[^:]*:Phoinix:6EDD:/
+89.5 "Brightened Fire #4" #sync / 1[56]:[^:]*:Phoinix:6EDD:/
+105.6 "Left Cinderwing/Right Cinderwing" sync / 1[56]:[^:]*:Phoinix:(66B4|66B5):/
+
+# There might be an enrage here? More data will be needed.
+110.7 "--untargetable--"
+118.0 "Trail of Condemnation" sync / 1[56]:[^:]*:Phoinix:66AD:/window 118.0,10
+127.0 "Blazing Rain" sync / 1[56]:[^:]*:Phoinix:66A2:/
```

Comment:
Adds are also targetable at this time point. May we add "--adds-targetable--" before?